### PR TITLE
Handled bad search queries & Fixed logo in navbar?

### PR DESCRIPTION
**Pull request by clee4037**

### CODE REVIEW

File: src/components/user-reservation/search/SearchBar.jsx [hunk 1]
```diff
@@ -51,17 +55,17 @@ const SearchBar = () => {
         <div className="join flex justify-center w-full">
           <input
             required
-            className="input input-bordered join-item pl-10 w-full"
+            className="input input-bordered focus:outline-none join-item pl-10 w-full"
             placeholder="Address, Place, City or Venue"
-            defaultValue=''
```
Comment (bzw24) [inline src/components/user-reservation/search/SearchBar.jsx:56]: lol the single to double quotes changed

**Review by justcord10 — APPROVED** [review]
Nice fix, looks nice

**Review by stevkim — APPROVED** [review]
Thanks for fixing the logo!

Also, great of you add in some error handling

**Review by bzw24 — APPROVED** [review]
Good with the navbar changes, thanks for investigating the logo fixed.

**Pull request merged**